src/index.html

```diff
@@ -1,23 +1,37 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>WebPack template</title>
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Capstone 2</title>
+  </head>
+  <body>
     <header>
-
+      <img class="logo" src="./assets/logo-movies.png" alt="logo" />
+      <nav class="menu">
+        <ul>
+          <li><a href="#!">Shows</a></li>
+        </ul>
+      </nav>
     </header>
 
     <main>
-        <h1>WebPack template</h1>
-               
+        <div>content</div>
     </main>
 
-<link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400&family=Raleway:wght@200;300;500;600&display=swap" rel="stylesheet">
-<script src="https://kit.fontawesome.com/4e5bc16f7d.js" crossorigin="anonymous"></script>
-</body>
-</html>
\ No newline at end of file
+    <footer>
+        Created by <a href="https://github.com/LuisDavidRodriguez" target="_blank">Luis Valades</a> and <a href="https://github.com/limavfabio" target="_blank">Fábio Lima</a> under MIT license
```

Comment:
Thank you my friend it looks nice! we only have a little problem with the linters because when wu add target=_blank we shall add the attribute rel="noopener"

Reply:
I'm going to fix this right now